db/migrations/0021.add_lti_type.py

```diff
@@ -0,0 +1,23 @@
+'''
+Add in the initial LTI Type Table
+'''
+
+from yoyo import step
+
+__depends__ = {'0020.add_lti_usage'}
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_type (
+            canvas_id BIGINT NOT NULL,
+            des VARCHAR(200),
+            rpt_group_nm VARCHAR(200),
+            PRIMARY KEY (canvas_id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT INTO lti_type (canvas_id, des, rpt_group_nm)
+        VALUES (25194, 'Zoom', 'Zoom'),
+               (4352, 'Bluejeans', 'Bluejeans')
+    '''),
```

Comment:
Yeah, we could just manually populate these without the migration but it would probably be nice to keep the migration files up-to-date if we ever do this.